### DISCUSSION TITLE
GPS Activities - Details in map view

### DISCUSSION THREAD

**Discussion by markusw1** [jlai/fitness-dashboard#40] [Ideas]
It would be fine if we can change from elapsed time (instead of clocktime) to elapsed distance in the map view.
Is a grafic/scale of the tempo possible ?
Also i miss the total altitude gain .
![Screenshot_22](https://github.com/user-attachments/assets/673d2d7d-40ca-4215-b145-73a47ea65dc6)

**Comment by jlai** [comment]
Yes, it's possible, it'll just take some work.

**Comment by jlai** [comment]
I've added an option to view the charts by elapsed distance. Let me know if you see any issues since distance is more complex than time (e.g. you could be standing still for 10 minutes and have 10 minutes worth of heart rate data points without traveling any distance). Will look into adding elapsed time (separate from clock time) in the future.

I've added elevation gain, although I've noticed that sometimes the Fitbit API reports 0 for elevation gain even when there's obviously been a change in elevation.

Will look into pace graphs in the future.

**Comment by markusw1** [comment]
Thank you - looks fine !
